javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -84,6 +84,280 @@ private API() {
     }
 
     //
+    // Aliases for static factories
```

Comment:
Interesting :). You're using `//` for delimiting groups in the same class, right?